src/BuildScriptGenerator/BaseBashBuildScriptProperties.cs

```diff
@@ -25,7 +25,7 @@ public class BaseBashBuildScriptProperties
         /// Gets or sets a list of OS packages that shuld be installed for this build.
         /// </summary>
         [NotNull]
-        public string[] OsPackagesToInstall { get; set; } = new string[0];
+        public string[] SystemPackagesToInstall { get; set; } = new string[0];
```

Comment:
What do you guys think of the name? We have been calling these 'system packages', so I changed all the places (switch name, environment variable name etc.) to keep it consistent.